.github/actions/cargo-build-macos-binary/action.yml

```diff
@@ -25,11 +25,12 @@ runs:
     - name: Install protoc
       run: brew install protobuf
       shell: bash
-    - uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        targets: ${{ inputs.target }}
-        override: true
```

Comment:
@guilload now that I think about it, why were we even overriding here in the first place? Doesn't it make more sense to build the releases with the toolchain versions pinned in `rust-toolchain.toml`?

Reply:
I don't know :) There was a time when we did not have a `rust-toolchain.toml` file. It should be the source truth indeed.